File: OrganizationService/src/main/java/com/slavacom/organizationservice/dto/old/OrganizationResponse.java
package com.slavacom.organizationservice.dto.old;

import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.AllArgsConstructor;
import lombok.Builder;

import java.time.Instant;
import java.util.List;
import java.util.UUID;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class OrganizationResponse {

    private UUID id;
    private String name;
    private String description;
    private UUID accountable;
    private boolean isActive;
    private Instant createdAt;
    private Instant updatedAt;

    // Дополнительная информация
    private long employeesCount;
    private long projectsCount;
    private List<ProjectTagsDto> availableTags;
}

File: OrganizationService/src/main/java/com/slavacom/organizationservice/dto/OrganizationResponse.kt
package com.slavacom.organizationservice.dto

import java.time.Instant
import java.util.*

data class OrganizationResponse(
    var id: UUID,
    var name: String,
    var description: String?,
    var accountable: UUID,
    var isActive: Boolean,
    var createdAt: Instant? = Instant.now(),
    var updatedAt: Instant?,
    var profileId: UUID? = null
)
